restui/models/mappings.py

```diff
@@ -162,6 +165,50 @@ def difference(self):
         
         return None
 
+    def statuses(self, usernames=False):
```

Comment:
I'd give this method a slightly different name, e.g. status_history, to better reflect what it does.

Reply:
As long as you don't mind plural. status_history is already the name of the reverse mapping for the foreign key relationship.

Reply:
I don't, but that would not make a difference as long as you add parentheses it will be parsed as a method call.

Reply:
True, but it just feels clearer if it has a different name.